test/unit/visual/cases/type.js

```diff
@@ -18,100 +18,10 @@ visualSuite("Typography", function () {
       p5.text("test", 0, 0);
       screenshot();
     });
-
-    visualTest('with a Google Font URL', async function(p5, screenshot) {
```

Comment:
It looks like this deleted some earlier tests. Would you be able to restore these?